engine-rest/engine-rest/pom.xml

```diff
@@ -686,34 +666,6 @@
       </dependencies>
       <build>
         <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <configuration>
-              <excludes>
-                <exclude>**/TaskRestServiceInteractionTest.java</exclude>
-                <exclude>**/NoServletEmptyBodyFilterTest.java</exclude>
-                <exclude>**/ServletEmptyBodyFilterTest.java</exclude>
-              </excludes>
-              <redirectTestOutputToFile>true</redirectTestOutputToFile>
-            </configuration>
-            <executions>
-              <execution>
-                <id>encoding-test</id>
-                <phase>test</phase>
-                <goals>
-                  <goal>test</goal>
-                </goals>
-                <configuration>
-                  <argLine>-Dfile.encoding=ISO-8859-1</argLine>
-                  <includes>
-                    <include>**/ProcessDefinitionRestServiceInteractionTest.java</include>
-                    <include>**/ResteasyTaskRestServiceInteractionTest.java</include>
-                  </includes>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
```

Comment:
Let's leave the `encoding-test` execution as discussed in https://github.com/camunda/camunda-bpm-platform/pull/2011

Reply:
I might get merge conflicts anyways which would have surfaced this, but I moved it over.